### ISSUE TITLE
docs(Icon): "Show similar" does not filter at all on the deployed docs site

### ISSUE THREAD

**Issue by patrikmolsson** [Semantic-Org/Semantic-UI-React#2193]
### Steps
Go to https://react.semantic-ui.com/elements/icon, have the "Show similar" checked (it is by default), and search for "home".

#### Environment
Have tested in Firefox 56.0.1 and Chrome 61.0.3163.100.

### Expected Result
The icon "home" will show up (and maybe some related icons)

### Actual Result
No filtering happens at all, for no search strings. It does however filter when **not** having the "Show similar" checked. 

It does work on the local build, however. I.e. when running ```yarn start``` in the source directory and accessing it from localhost, the "Show similar" functionality works.

**Comment by layershifter** [comment]
@patrikmolsson thanks for report, any investigation of this problem is much appreciated

**Comment by patrikmolsson** [comment]
@layershifter I can look into it!

**Comment by patrikmolsson** [comment]
It seems like it was disabled by design in one of the library files, see https://github.com/Semantic-Org/Semantic-UI-React/blob/0a29f6c62ab700d232a841356023223df8d4d01a/src/lib/leven.js#L7 
Although, I removed that line on [my deployed pages](https://patrikmolsson.github.io/Semantic-UI-React/), and it seems to work OK.

**Comment by patrikmolsson** [comment]
What are your thoughts on this, @layershifter ? Can you think of any reason why this line should be there?

I found another usage of ``leven``, and it seems to be used in 
https://github.com/Semantic-Org/Semantic-UI-React/blob/0a29f6c62ab700d232a841356023223df8d4d01a/src/lib/customPropTypes.js#L21

What is the usage of that function?

**Comment by layershifter** [comment]
@patrikmolsson Thanks for your research :+1: The `suggest` function is one of `customPropTypes`, their usage will be also removed in the production build.

I'm think that the better solution is duplicate the `leven` function if `docs/app`, because our users don't need it in their prodution builds. Do you want to make the PR?

**Comment by patrikmolsson** [comment]
@layershifter Sure, I can do it!

However, I don't really follow. If the customPropTypes are removed in the production build, then the if statement in the leven.js file have no effect, right? Then we could just remove it?

**Comment by levithomason** [comment]
There are two bundles in this repo, the `/docs` and the `/src`.  Webpack is responsible for bundling all the dependencies for these, for which we have separate build scripts.

We include a custom copy of `leven` in the source code because we needed to ensure it was removed from the production build of the library.  This is because it is only used to find related prop values.  Prop types are removed from our production bundle, so we don't need to include `leven` in the production build either.

However, the docs do not have this limitation.  They actually use `leven` in the production docs app, for similar icon search.  In this case, we can just `yarn add leven --dev` and only import it in the docs.  Then, instead of importing `leven` from `src/lib`, we can import it directly from node_modules `import leven from 'leven'`.

Now, the doc site will always include a bundled version of leven.
